src/types/mod.rs

```diff
@@ -2031,6 +2031,10 @@ pub enum DelegationEvent {
         /// Delegator's id
         delegator_id: DelegatorId,
     },
+    BakerRemoved {
+        /// Baker's id
+        baker_id: BakerId,
```

Comment:
Is the ID still valid after the removal? Perhaps the comment could touch on this (as it is, the comment adds nothing).